Colab_DAIN.ipynb

```diff
@@ -94,6 +93,10 @@
         "#@markdown A path, relative to your GDrive root, where you want the generated frame.\n",
         "FRAME_OUTPUT_DIR = '/content/DAIN/output_frames' #@param{type:\"string\"}\n",
         "\n",
+        "#@markdown ## Frame mixer\n",
+        "#@markdown Normally we interpolate between two adjacent frames. You can try to experiment with other mixers. Claymation mixer can give smoother outputs when animation only moves every second frame at times.\n",
+        "FRAME_MIXER = \"normal\" #@param [\"normal\", \"claymation\"] {type:\"string\"}\n",
```

Comment:
I think greased is missing here:

```suggestion
        "FRAME_MIXER = \"normal\" #@param [\"normal\", \"claymation\", \"greased\"] {type:\"string\"}\n",
```

Reply:
I found greased to be useless, its results disappointing. Didn't want to pollute the colab API with that. It may be best to remove it completely, but the maintenance burden is small, so left it in.

Reply:
Sounds good to me! I'm glad it wasn't a mistake.